ext/php7/serializer.c

```diff
@@ -1058,7 +1058,7 @@ static void signalfx_serialize_sfx_span_to_array(zval* spans_array, ddtrace_span
         zval *dd_service = zend_hash_str_find(Z_ARR_P(dd_span), ZEND_STRL("service"));
 
         if (dd_service != NULL && Z_TYPE_P(dd_service) == IS_STRING) {
-            add_assoc_zval(tags, SFX_TAG_COMPONENT, dd_service);
+            _add_assoc_zval_copy(tags, SFX_TAG_COMPONENT, dd_service);
```

Comment:
What's the reason for this change (if it even differs, assume the first method does not copy data)?